test/files/nth-fragment/table.html

```diff
@@ -0,0 +1,303 @@
+        }
+        @-epubx-page-template {
+            .partition {
+                width: 398px;
+                height: 198px;
+                border: rgba(0, 0, 255, 0.2) solid 1px;
+                -epubx-flow-from: body;
+            }
+            .left {
+                left: 10px;
+            }
+            .right {
+                right: 10px;
+            }
+
+            @-epubx-page-master {
+                @-epubx-partition {
+                    bottom: 0;
+                    left: 0;
+                    right: 0;
+                    height: 10px;
+                    font-size: 10px;
+                    line-height: 10px;
+                    text-align: center;
+                    content: counter(page);
+                }
+                @-epubx-partition class(partition) class(right) {
+                    top: 610px;
+                }
+                @-epubx-partition class(partition) class(right) {
+                    top: 410px;
+                }
+                @-epubx-partition class(partition) class(right) {
+                    top: 210px;
+                }
+                @-epubx-partition class(partition) class(right) {
+                    top: 10px;
+                }
+                @-epubx-partition class(partition) class(left) {
+                    top: 610px;
+                }
+                @-epubx-partition class(partition) class(left) {
+                    top: 410px;
+                }
+                @-epubx-partition class(partition) class(left) {
+                    top: 210px;
+                }
+                @-epubx-partition class(partition) class(left) {
+                    top: 10px;
+                }
+            }
+        }
+        :root {
+            font-size: 10px;
+            line-height: 12px;
+            widows: 1;
+            orphans: 1;
+        }
+        body {
+            margin: 0 10px;
+        }
+        section {
+            break-after: column;
+        }
+        h1 {
+          font-size: 10px;
+          font-weight: normal;
+        }
+
+        table {
+            border: 2px black solid;
+            border-spacing: 2px;
+        }
+        tr, .tr {
+            border: 1px blue solid;
+        }
+        th, td, .th, .td {
+            border: 1px gray solid;
+            line-height: 16px;
+            vertical-align: top;
+            width: 160px;
+        }
+        thead, .thead {
+            background-color: #FFAAAA;
+        }
+        tfoot, .tfoot {
+            background-color: #AAAAFF;
+        }
+
+        table::nth-fragment(2n) {
+            border: 2px red solid;
+        }
+        tr::nth-fragment(2n) td {
+            border: 1px blue solid;
+        }
+        td::nth-fragment(2n) {
+            color: red;
+        }
+        td::nth-fragment(3) {
+            color: blue;
+        }
+        div.cell_content::nth-fragment(2n) {
+            font-weight: bold;
+        }
+
+        table2.td {
```

Comment:
fix it in [this commit](https://github.com/vivliostyle/vivliostyle.js/pull/342/commits/1bd56ac2bb6afe5ad68dbdd4256bc657ec09da0a).